classes.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+#define PLUS     0
+#define MINUS    1
+#define MULTIPLY 2
+#define DIVIDE   3
+
+class coordinate {
+private:
+    int x;
+    int y;
+
+public:
+    coordinate() : x(0), y(0) {}
+    coordinate(int x, int y) : x(x), y(y) {}
+
+    int getX() { return x; }
```

Comment:
Please, rewrite second "} " in a next line and add an indents.

Reply:
fixed